packages/fxa-payments-server/src/components/PaymentConfirmation/index.scss

```diff
@@ -3,18 +3,20 @@
 
 .payment-confirmation {
   background: $color-white;
-  border: 1px solid  #e7e7e7;
+  border: 1px solid #e7e7e7;
   border-bottom: 0;
-  border-radius: 8px;
-  display: flex;
-  flex: 1;
-  flex-direction: column;
-  font-size: 15px;
-  height: 100%;
-  margin: 0;
-  max-width: 600px;
-  min-width: 375px;
+  color: rgba(12, 12, 13, 0.8);
+  min-height: 100%;
   padding: 20px 16px 0;
+  width: 100%;
+
+  @include min-width('tablet') {
+    border-top: 0;
+  }
+
+  @include min-width('desktop') {
+    margin: -24px 0;
```

Comment:
Same comment here as below, let's adjust the padding instead of moving the entire section "up" with negative margin.